com.unity.render-pipelines.universal/Runtime/Passes/DebugPass.cs

```diff
@@ -0,0 +1,115 @@
+namespace UnityEngine.Rendering.Universal.Internal
```

Comment:
We should avoid duplicating code and this looks very similar to FinalBlitPass already. Maybe we can make the necessary adjustments to FinalBlitPass to support this.

Reply:
I'll take a look

Reply:
Debug pass no longer exists - it's not dealt with by the various final passes and their shaders.